index.html

```diff
@@ -25,6 +26,14 @@
         <li><a href="#about-section">About</a></li>
         <li><a href="#contact">Contact</a></li>
       </ul>
+      <div id="theMenu">
+        <ul class="nav-items2">
+            <li><a class='navs' href="#work-section">Portfolio</a></li>
+            <li><a class='navs' href="#about-section">About</a></li>
+            <li><a class='navs' href="#contact">Contact</a></li>
```

Comment:
- [ OPTIONAL ] since the previous code reviewer doesn't mention that I think it would be better if you add more margin/padding to the top of the menu link since I think it is very close to the (X) button so please add some margin/padding 💯♻🔥✅

- [ OPTIONAL ] well done but I think it would be great if you decrease the font weight a little bit to the li elements since I think in your design the font weight is bigger than the Figma🙏🏻♻🔥


- Your design 
![image](https://user-images.githubusercontent.com/58553711/217116148-85add24a-2e8c-47d2-9ca8-8d04b904fc1a.png)


- Figma
![image](https://user-images.githubusercontent.com/58553711/217116218-7b792d48-3b8e-4d36-b6fb-094d3c7050ea.png)